src/components/get-started-card/index.tsx

```diff
@@ -14,9 +15,18 @@ function GetStartedCard({
 	body,
```

Comment:
It was discussed that we may want to rename this to be more generic, given the Vagrant designs. I will wait to do that until the vagrant work.